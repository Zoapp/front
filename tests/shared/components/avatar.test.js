import React from "react";
import renderer from "react-test-renderer";
import Avatar from "shared/components/avatar";

// Avatar component is a simple props only components
//
describe("Avatar", () => {
  it("renders correctly", () => {
    const component = renderer.create(React.createElement(Avatar));

    const tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });

  it("can take an src and size prop", () => {
    const component = renderer.create(
      React.createElement(Avatar, {
        src: "http://www.gravatar.com/avatar/?d=identicon",
        size: 100,
      }),
    );

    const tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });
});
