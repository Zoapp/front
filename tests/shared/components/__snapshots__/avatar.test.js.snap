// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Avatar can take an src and size prop 1`] = `
Array [
  <i
    className="material-icons"
    style={
      Object {
        "fontSize": 100,
      }
    }
  >
    account_circle
  </i>,
  <img
    className=""
    onError={[Function]}
    onLoad={[Function]}
    src="http://www.gravatar.com/avatar/?d=identicon?size=100"
  />,
]
`;

exports[`Avatar renders correctly 1`] = `
<i
  className="material-icons"
  style={
    Object {
      "fontSize": 48,
    }
  }
>
  account_circle
</i>
`;
